test/test_images/probe_helper/probe/handlers/cloudpubsub.go

```diff
@@ -70,6 +72,7 @@ func (p *CloudPubSubSourceProbe) Forward(ctx context.Context, event cloudevents.
 		return fmt.Errorf("CloudPubSubSource probe event has no '%s' extension", topicExtension)
 	}
 	ctx = cecontext.WithTopic(ctx, fmt.Sprint(topic))
+	logging.FromContext(ctx).Infow("Publishing message to pubsub topic", zap.String("topic", fmt.Sprint(topic)))
```

Comment:
I don't think you need `Sprint`, did you try not having it? Worst case you can do `topic.String()` if that does not work.

Reply:
`topic` is of type `interface{}`, it does not implement the `String` method. So `fmt.Sprint` just performs that cast. But I should probably have used `topic.(string)` instead, I will keep this in mind in the future.